projects/igniteui-angular/src/lib/select/select-navigation.directive.ts

```diff
@@ -53,7 +53,7 @@ export class IgxSelectItemNavigationDirective extends IgxDropDownItemNavigationD
             }
         }
 
-        if (!this.target.collapsed && (key === 'tab')) {
+        if (!this.target.collapsed && (key === 'tab' || (event.shiftKey && key === 'tab'))) {
```

Comment:
So either a tab (with or w/o shift) or a tab w/ shift? Isn't it all the same then?